modules/nf-core/nacho/normalise/main.nf

```diff
@@ -0,0 +1,58 @@
+nextflow.enable.moduleBinaries = true
+
+process NACHO_NORMALISE {
+    tag "$sample_sheet"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container 'community.wave.seqera.io/library/r-dplyr_r-fs_r-ggplot2_r-nacho_pruned:033bc017f5f36b6d'
+
+    input:
+    path rcc_files, stageAs: "input/*"
+    path sample_sheet
+
+    output:
+    path "*normalized_counts.tsv"          , emit: normalized_counts
```

Comment:
Why the * in `normalized_counts.tsv`? Isn't the output always going to be `normalized_counts.tsv`? Or do you expect a `sample_id` could be added?

Reply:
I took that from the original module, but I was also speculating with adding a prefix there